source/_integrations/xiaomi_miio.markdown

```diff
@@ -491,6 +493,38 @@ Buzzer                  | Turn on/off the buzzer
 Child Lock              | Turn on/off the child lock
 LED                     | Turn on/off the LED
 
+### Air Fresh A1 (dmaker.airfresh.a1)
+
+- Power (on, off)
+- Operation modes (Auto, Sleep, Favorite)
+- Sensor entities
+
+Sensor                          | Description                                                    | Enabled by default
+------------------------------- | -------------------------------------------------------------- | -----------------------
+Carbon Dioxide                  | The current carbon dioxide measured in ppm                     | True
+Dust filter life remaining      | The remaining life of the filter                               | True
+Dust filter life remaining days | Filter usage time in day                                       | True
+PM2.5                           | The current particulate matter 2.5 measured                    | True
+Temperature                     | The current outside temperature measured                       | True
+Control Speed                   | The current motor speed measured in rpm                        | True
+Favorite Speed                  | The favorite motor speed measured in rpm                       | True
+Ptc status                      | The heater status                                              | True
+
+- Switch entities
+
+Switch                  | Description
+----------------------- | -----------------------
+Buzzer                  | Turn on/off `buzzer`
+Child Lock              | Turn on/off `child lock`
+Display                 | Turn on/off `display`
+Ptc                     | Turn on/off `heater`
+
+- Button entities
+
+Button                  | Description                                | Enabled by default
+----------------------- | ------------------------------------------ | ---------------------
+Reset dust filter       | Resets filter lifetime of the dust filter  | False
```

Comment:
Is there some specific reason not to enable this by default? I have oftentimes seen users being confused of sensors not being available for no good reason, a good reason in my opinion being to save I/O.

Reply:
For me it's more like hide "dangerous" button from user. But if a main reason must be I/O I`ll change this when provide button PR to core.

Reply:
So I don't know what's the official stance on this, I just personally feel that entities should not be hidden from users for no obvious reasons. If it's really that dangerous, it probably should not be accessible at all.

Reply:
> official stance on this

Entities should not be disabled by default unless they are high frequently updating.
As this is a button... there is no updating at all.

This is also documented as such in our developer documentation.

Ergo: These entities should not have been disabled by default.

Reply:
Remove "enabled by default"  for button but not for sensors  to have the same struct with other device like "Air Purifier 3C" etc.